src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -248,6 +248,16 @@ export const createRequestSignature = (
     };
 };
 
+/**
+ * Generate a schnorr signature using the group's private key over the group's provided id
+ * @param groupPrivateKey
+ * @param groupPublicKey
+ * @param groupId
```

Comment:
If we're not going to comment on what these params are (which I don't think is really necessary) they should just be removed from the doc block. In this form they don't help anybody.

Reply:
done